plugins/modules/mso_schema.py

```diff
@@ -18,18 +20,29 @@
 - Manage schemas on Cisco ACI Multi-Site.
 author:
 - Dag Wieers (@dagwieers)
+- Akini Ross (@akinross)
 options:
   schema:
     description:
     - The name of the schema.
     type: str
     aliases: [ name ]
+  id:
+    description:
+    - The id of the schema.
+    - This parameter is required when the C(schema) needs to be updated.
+    type: str
+  description:
+    description:
+    - The description of the schema.
+    type: str
   state:
     description:
     - Use C(absent) for removing.
     - Use C(query) for listing an object or multiple objects.
+    - Use C(present) for creating or updating. Only supported on versions of MSO that are 4.1 or greater.
     type: str
-    choices: [ absent, query ]
+    choices: [ absent, query, present ]
     default: query
 notes:
 - Due to restrictions of the MSO REST API this module cannot create empty schemas (i.e. schemas without templates).
```

Comment:
Can we change this statement to reflect the behaviour in newer versions of MSO?

Reply:
changed